package.json

```diff
@@ -15,7 +15,7 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.20",
+    "@docusaurus/core": "^2.3.0",
     "@docusaurus/plugin-client-redirects": "^2.0.0-beta.21",
```

Comment:
You need to update all of the packages at once, not just `core`

Reply:
aha, thanks! done.